ai/agentic-ui-generator/getting-started.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting Started
+page_title: Getting Started with the Telerik UI for Blazor Agentic UI Generator
+description: Learn how to install and configure the Telerik UI for Blazor Agentic UI Generator in Visual Studio and Visual Studio Code.
+slug: agentic-ui-generator-getting-started
+position: 10
+tags: telerik,blazor,ai,agentic,ui,generator,installation
+published: True
+---
+
+# Getting Started with the Agentic UI Generator
+
+This article describes how to install, configure, and use the Telerik UI for Blazor Agentic UI Generator in Visual Studio and Visual Studio Code.
+
+## Prerequisites
+
+An AI-powered IDE that supports MCP servers, for example, Visual Studio, Visual Studio Code, Cursor.
+
+## Installation
+
+The Agentic UI Generator is available as an MCP (Model Context Protocol) server that integrates with AI-powered IDEs like Visual Studio Code, Cursor, and other MCP-compatible tools.
+
+### Configuration Steps
+
+This section contains information about how to set up the Agentic UI Generator in Visual Studio and Visual Studio Code.   
+
+#### Visual Studio
+
+1. Log in with a valid [Telerik user account](https://www.telerik.com/account/).
```

Comment:
This lacks proper context. Where should people log in? In Visual Studio or somewhere else? Consider the following change:


```suggestion
1. In your browser, log into your [Telerik account](https://www.telerik.com/account/).
```